GlimpseImputationPipeline/glimpse_docker/build_base_and_extension_docker.sh

```diff
@@ -2,7 +2,7 @@
 
 # This branch is using the following configuration.
 # To replicate this branch, run this script with the following arguments:
-# build_base_and_extension_docker.sh -r "https://github.com/kachulis/GLIMPSE.git" -b ck_checkpoint_clean -t "glimpse_build_test"
+# build_base_and_extension_docker.sh -r "https://github.com/odelaneau/GLIMPSE.git" -b master 
```

Comment:
Sorry, really nitpicky, but the script requires the `-t` argument to be set. Maybe just add it here for good measure